Poker.BE/Poker.BE.Domain/Game/Hand.cs

```diff
@@ -22,13 +22,16 @@ public class Hand
         private Deck deck;
         private ICollection<Player> activePlayers;
         private Pot pot;
+        private Player dealer;
         #endregion
 
         #region Properties
+        public bool Active { get; set; }
```

Comment:
we need to validate the pre condition "The previous hand has ended. (if it’s the first hand ignore this part)", so this is the best option here IMO.